syntax = "proto3";

package proto;

service BookingService {
    rpc getAllFlights(getAllFlightsRequest) returns (getAllFlightsReply);
}

message getAllFlightsRequest {
    int32 giorno = 1;
    int32 mese = 2;
    int32 anno = 3;
    string aereoporto_arrivo = 4;
    string aereoporto_partenza = 5;
    int32 persone = 6;
    
}

message getAllFlightsReply {
	string id = 1;
	string compagnia = 2;
}
